app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,15 @@
+class CommentsController < ApplicationController
+  def create
+    @job = Job.find(params[:job_id])
+    @comment = @job.comments.create(comment_params)
+
+    redirect_to company_job_path(@job.company_id, @job)
```

Comment:
`@job.company` and `@job.company_id` are the same FYI. I like referencing without id but its your preference